lib/utils/validator.dart

```diff
@@ -45,10 +45,14 @@ class Validator {
   // }
 
   static String validatePassword(String value) {
+    String pattern =
+        r'^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[!@#\$&*~]).{8,}$';
+    RegExp regExp = new RegExp(pattern);
     if (value.length < 6) {
       return 'Password must be at least 6 characters.';
+    } else if (!regExp.hasMatch(value)) {
+      return 'Password must atleast 1 A-Z,a-z,0-9&special character';
```

Comment:
User won't understand what is meaning of this.
Please show meaningful message, toast or suggestion.

Reply:
can u give some suggestion plz @Sagar2366 
it will be really helpful

Reply:
what should be printed instead?

Reply:
If you can show toast/message box which can clearly show why the error that would be great addition.
Sample : https://ux.stackexchange.com/questions/108307/a-clear-error-message-for-wrong-password-format

Reply:
plz see i have updated it if it is not workable i will change it

Reply:
thanks for suggestion

Reply:
@shikharsaurav while entering password only you can show toast, should give idea which criteria is matched and which are still left.
sample - 
<img width="505" alt="Screenshot 1942-12-28 at 6 36 05 PM" src="https://user-images.githubusercontent.com/26805882/111784993-0ef73300-88e2-11eb-805b-d2bbe3d11f22.png">

Reply:
ok sir i will do it like that